bundles/org.openhab.binding.airvisualnode/src/main/java/org/openhab/binding/airvisualnode/internal/discovery/AirVisualNodeDiscoveryService.java

```diff
@@ -63,20 +67,23 @@ protected void startScan() {
     @Override
     protected void startBackgroundDiscovery() {
         logger.debug("Starting background discovery");
-        backgroundDiscoveryFuture = scheduler.scheduleWithFixedDelay(this::scan, 0, 5, TimeUnit.MINUTES);
+        ScheduledFuture<?> localDiscoveryFuture = backgroundDiscoveryFuture;
+        if (localDiscoveryFuture == null || localDiscoveryFuture.isCancelled()) {
+            backgroundDiscoveryFuture = scheduler.scheduleWithFixedDelay(this::startScan, 30, REFRESH_MINUTES,
```

Comment:
Did you mean to change `initialDelay` from 0 to 30 minutes and `delay` from 5 to 600 minutes?

Reply:
No this was not meant to be the case, i accidently grabbed the 600 from the singleto at line 58 i guess. Thanks you saved me here :-)